docs/planning/releaseplanning.md

```diff
@@ -0,0 +1,39 @@
+# Knative Eventing Release Planning - Proposal
+
+This document summarizes release planning for Knative Eventing.
+
+Releases and sprints are both represented in Github as Milestones. Releases
+span 6 weeks, starting on a Wednesday and ending on a Tuesday 6 weeks later.
+Releases are further broken down into 3 2-week sprints.
+
+Each work group (WG) will be responsible for implementing this process.
+
+![release planning overview](images/releaseplanning.png)
+
+## Recommended process
+
+No later than midway through a release, start planning for the next release:
+- As before, anyone can create new issues for Feature Requests or Bugs.
+- Label issues to propose them for a particular release. For example:
+proposed/0.13, proposed/0.14.
+- Community votes on issues anytime with thumbs-up or thumbs-down.
+
+One week before release:
+- WG leads and representatives from each contributing vendor will meet and
```

Comment:
Should we talk at some point about what the resource allocations look like? Otherwise, it will be tricky to figure out what's reasonable and starting with a list of things that's untenable seems like a bad starting point? :)